### PR TITLE
Fix/hoomd2.6

### PR DESCRIPTION
**Pull request by astatt**
This fix removes the name clash described in Issue #9 Support HOOMD 2.6.0. Minimum HOOMD version is bumped to HOOMD 2.6.0 as well.

### CODE REVIEW

File: azplugins/BounceBackNVEGPU.cu [hunk 1]
```diff
@@ -10,7 +10,10 @@
  */
 
 #include "BounceBackNVEGPU.cuh"
-#include "BounceBackGeometry.h"
```
Comment (mphoward) [inline azplugins/BounceBackNVEGPU.cu]: I'm in favor of leaving the mpcd headers in `BounceBackGeometry.h` and not explicitly including them individually here, so you can revert this change. Otherwise, remove the headers from `BounceBackGeometry.h`.

File: azplugins/CMakeLists.txt [hunk 2]
```diff
@@ -15,6 +15,10 @@ endif(NOT DEFINED HOOMD_EXTERNAL_BUILD)
 if (NOT BUILD_MD)
     message(FATAL_ERROR "azplugins requires hoomd installation with MD component.")
 endif (NOT BUILD_MD)
+# the md component is always required
```
Comment (mphoward) [inline azplugins/CMakeLists.txt]: `md` -> `mpcd`

File: azplugins/module.cc [hunk 3]
```diff
@@ -55,6 +55,10 @@ namespace py = pybind11;
 
 /* Integrators */
 #include "BounceBackGeometry.h"
+#ifdef ENABLE_MPCD
```
Comment (mphoward) [inline azplugins/module.cc]: Again, if you stick these in `BounceBackGeometry.h`, no need to include here.

File: azplugins/BounceBackNVEGPU.cu (hunk 1 above)
Comment (astatt) [reply azplugins/BounceBackNVEGPU.cu]: I've put the headers in `BounceBackGeometry.h` and removed them from  `BounceBackNVEGPU.cu` and module.cc .
